lametro/templates/base.html

```diff
@@ -110,6 +110,20 @@
 
     {% block extra_js %}{% endblock %}
 
+    <script type="text/javascript">
+        $('label.btn').click(function toggleRadio(e) {
```

Comment:
Since this is on the base template, is using `label.btn` as a selector going to have unintended consequences on some pages?